### DISCUSSION TITLE
Change of the background color for author highlighting after upgrade to v5

### DISCUSSION THREAD

**Discussion by gh-andre** [gitextensions/gitextensions#12059] [General] [answered]
I upgraded Git Extensions on two computers and before the upgrade author highlight color was light yellow on both systems (right side). After the upgrade, the highlight on one of the systems changed to light blue (left side). Both are running the same version of Git Extensions  - 5.1.1.

![gitext-author-bg-color-change](https://github.com/user-attachments/assets/3296f09d-1949-49a6-b8a6-95fbfa1e9464)

Both systems use the same default theme and `invariant.css` is binary-same in both cases. The only difference I can see is that the system on the left (blue) is running Windows 11 and the one on the right is running Windows 10.

Does anybody have any idea what I can change to get back that yellow background? Thanks.

**Comment by mstv** [comment, marked as answer]
This was changed in #11748.
`invariant.css` - or a copy of it - is the right place for an adaptation.
I guess the installation on your Win10 machine uses another scheme - most likely `migrated.css` yet.

![image](https://github.com/user-attachments/assets/12b4bbba-8ea1-46f8-9e70-cef7beaf1551)

**Reply by gh-andre** [reply]
I always used the default theme and only checked that the app theme folder didn't change, and didn't realize that Git Extensions automatically switched to the migrated theme on the Win10 machine. Thank you for pointing it out.

Any idea why it would create a migrated theme in one case, but not in the other?

**Reply by mstv** [reply]
In early versions, colors were stored in the settings file. These are migrated once if an old settings file is found.

**Reply by gh-andre** [reply]
I see. Good to know. Thank you for your help. It is much appreciated.
